_posts/General/2024-01-26-terms-and-acronyms/2024-01-26-terms-and-acronyms.md

```diff
@@ -18,36 +18,36 @@ sidebar: []
 **FMCB** = *"Free MemoryCard Boot"* -->
 - This is the most commonly used exploit/Softmod used on a vast variety of models.
 - It is being installed on the MemoryCard.
-- Works on SCPH-10000 to SCPH-90xxx.
+- Works on SCPH-10000 to SCPH-90xxx (depending on the program version).
 - Some 90k have BOOT-ROM 2.30 however and won't automatically boot FMCB.
 
 **FHDB** = *"Free HardDrive/HardDisk Boot"* -->
 - internal storage/HDD variation of FMCB for FAT PS2s as well as 70k-Slim PS2s!
 - It is installed onto the internal HDD.
-- Works on 10k to 70k! (Note: 10k, 15k and 18k consoles also need a memory card update to boot from hdd)
```

Comment:
not sure wtf happened here. but this is gone for some reason. @koraxial you might wanna check it

Reply:
It got deleted with your commit, I assumed it was intentional so I merged it. Will fix now...